### ISSUE TITLE
bug: $ionicPlatform.ready not firing on Android when app delivered over the web

### ISSUE THREAD

**Issue by ranjitpandit** [driftyco/ionic#3902]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>android</span> <span ionic-platform-version>5.1</span>  <span ionic-webview>webview</span>

<span ionic-description>I'm trying to deliver my Ionic hybrid app over the web so in config.xml I have pointed to index.html on my web server
 <content src="https://www.webservername/index.html"/>
The web delivered app works great on iOS but the app is not starting up on Android. 
I experimented this case with the default "starter" app that is part of the ionic framework and I'm seeing the same behavior. 
It turns out that when the app is delivered over HTTP the $ionicPlatform.ready event is NOT firing on Android.
The events fires properly on iOS though.

I have tried numerous experiments but no success. 

To summarize:
Local version of app runs fine on both iOS and Android
Remote(http) version of the app runs fine only on iOS but fails on Android.

Android version: 5.1.1
Cordova version: 5.0.0
Ionic Version: 1.0.0</span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @ranjitpandit!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=3902). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=3902)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by ranjitpandit** [comment]
I get this in the logs when the app is loaded from the webserver.

deviceready has not fired after 5 seconds.
cordova.js:1178 Channel not fired: onCordovaInfoReady

**Comment by ranjitpandit** [comment]
any ideas?

**Comment by Ionitron** [comment]
Greetings @ranjitpandit!

I've closed this issue because my sensors indicated it was old and inactive, and may have already been fixed in recent versions of Ionic. However, if you are still experiencing this issue, please feel free to reopen this issue by creating a new one, and include any examples and other necessary information, so that we can look into it further.

Thank you for allowing me to assist you.
